projects/aca-content/src/lib/components/info-drawer/library-metadata-tab/library-metadata-form.component.scss

```diff
@@ -1,5 +1,51 @@
+/* stylelint-disable selector-class-pattern */
+
 app-library-metadata-form {
   .app-library-metadata-form-field {
     width: 100%;
+
+    .mat-mdc-form-field-flex {
+      border-bottom: 1px solid;
+
+      .mat-mdc-form-field-infix {
+        padding-bottom: 3px;
+      }
+    }
+
+    &.mat-form-field-disabled {
+      .mat-mdc-form-field-flex {
+        border-bottom: 1px dotted rgba(0, 0, 0, 0.3);
+      }
+    }
+
+    &.mat-focused {
+      .mat-mdc-form-field-flex {
+        border-bottom: 2px solid rgb(0, 85, 184);
```

Comment:
Let's try to use variables for colors so it will be easier to reuse and manage between places